proto/penumbra/penumbra/core/component/funding/v1/funding.proto

```diff
@@ -116,10 +117,49 @@ message ActionLiquidityTournamentVoteView {
     shielded_pool.v1.NoteView note = 2;
   }
   message Opaque {
-    ActionLiquidityTournamentVote vote = 1;    
+    ActionLiquidityTournamentVote vote = 1;
   }
   oneof liquidity_tournament_vote {
     Visible visible = 1;
     Opaque opaque = 2;
   }
 }
+
+message EventLqtDelegatorReward {
+  // The epoch for which the reward was paid.
+  uint64 epoch_index = 1;
+  // The amount of reward, in staking tokens.
+  num.v1.Amount reward_amount = 2;
+  // The amount of reward, in delegation tokens.
+  asset.v1.Value delegation_tokens = 3;
+  // The recipient of the reward
+  keys.v1.Address address = 4;
+  // The incentivized asset.
+  asset.v1.AssetId incentivized_asset_id = 5;
+}
+
+message EventLqtPositionReward {
+  // The epoch for which the reward was paid.
+  uint64 epoch_index = 1;
+  // The amount of the reward, in staking tokens.
+  num.v1.Amount reward_amount = 2;
+  // The liquidity position receiving the reward
+  dex.v1.PositionId position_id = 3;
+  // The incentivized asset.
+  asset.v1.AssetId incentivized_asset_id = 4;
+  // The total volume for the pair during the tournament, in staking tokens.
+  num.v1.Amount tournament_volume = 10;
+  // The cumulative volume for the LP, in staking tokens.
+  num.v1.Amount position_volume = 11;
+}
+
+message EventLqtVote {
```

Comment:
I think we should add the recipient address and the transaction id. Tx IDs would be a good way to track "unique votes" roughly.